conf/messages.cy

```diff
@@ -469,7 +471,9 @@ technical.issue.heading=Mae’n ddrwg gennym, mae problem gyda’r gwasanaeth
 technical.issue.try.later=Rhowch gynnig arall arni yn nes ymlaen.
 technical.issue.back=Dychwelyd i GOV.UK
 technical.other-ways.h1=Ffyrdd eraill o wneud cais am Lwfans Priodasol
-technical.other-ways.para0=Gallwch wneud cais am Lwfans Priodasol drwy''ch <a href="https://www.gov.uk/cyfrif-treth-personol">Cyfrif Treth Personol</a>, o dan ‘gwasanaethau y mae''n bosibl y bydd eu hangen arnoch’.
+technical.other-ways.para0.part1=Gallwch wneud cais am Lwfans Priodasol drwy''ch
```

Comment:
Wrong type of quotes. The english message also includes an anchor tag, but this doesn't. Is that expected?

Reply:
I think its expected and the difference is due to incorrect English message

Reply:
https://smartquotesforsmartpeople.com/ should be used for quotes so copy & paste